src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age !== undefined ? (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
```

Comment:
checklist 1.[CODE STYLE] - Use destructuring for getting access to values of props object
instead of using `person.name` and getting values by keys, make smth like that 
`export const Component = (props) => {
 const {
  firstProperty,
  secondProperty,
  thirdProperty
} = props; 

 return (
  <>
    <h1>{firstProperty}</h1>
    <span>{secondPropery}</span>
    <p>{thirdProperty}</p>
  </>
 )`